build/Makefile

```diff
@@ -6,16 +6,22 @@ OUT_WA = "sqlite.wasm"
 OUT_BN = "sqlite.js"
 OUT_TY = "sqlite.d.ts"
 
+HOST_OS = linux
```

Comment:
Can we move this down to be with the if statement 😅

Reply:
You mean like the following?
```mk
ifeq ($(OS),Windows_NT)
    # ...
endif

ifeq ($(shell uname),Linux)
    # ...
endif

ifeq ($(shell uname),Darwin)
    # ...
endif
```